src-ui/app/edit-screen/components/mapping-pane/VariantDisplay.cpp

```diff
@@ -532,6 +532,9 @@ void VariantDisplay::rebuild()
     case dsp::InterpolationTypes::Linear:
         srcButton->setLabel("LIN");
         break;
+    case dsp::InterpolationTypes::ZeroOrderHoldAA:
+        srcButton->setLabel("ZAA");
```

Comment:
I would put `ZOH-AA` here, even if it doesn't fit now. The font in the value edit should be a bit smaller so it could fit (and maybe this area might get a bit wider so these controls would be also wider). That's all WIP still.

Reply:
Fair about the naming